docs/docs/config/behaviors.md

```diff
@@ -51,6 +51,13 @@ Creates a custom behavior that triggers one behavior when a key is held or a dif
 
 See the [hold-tap behavior](../behaviors/hold-tap.mdx) documentation for more details and examples.
 
+### Kconfig
+
+| Config                                             | Type | Description                                                                                   | Default |
+| -------------------------------------------------- | ---- | --------------------------------------------------------------------------------------------- | ------- |
+| `CONFIG_ZMK_BEHAVIOR_HOLD_TAP_MAX_HELD`            | int  | Maximum number of simultaneous held hold-taps.                                                | 10      |
+| `CONFIG_ZMK_BEHAVIOR_HOLD_TAP_MAX_CAPTURED_EVENTS` | int  | Maximum number of system events to capture while deferring a hold or tap decision resolution. | 40      |
```

Comment:
```suggestion
| Config                                             | Type | Description                                                                                  | Default |
| -------------------------------------------------- | ---- | -------------------------------------------------------------------------------------------- | ------- |
| `CONFIG_ZMK_BEHAVIOR_HOLD_TAP_MAX_HELD`            | int  | Maximum number of simultaneous held hold-taps                                                | 10      |
| `CONFIG_ZMK_BEHAVIOR_HOLD_TAP_MAX_CAPTURED_EVENTS` | int  | Maximum number of system events to capture while deferring a hold or tap decision resolution | 40      |
```
Nit, we don't seem to use periods in these typically. Although I see others that aren't consistent either, and can be fixed in the future.